### ISSUE TITLE
Add support for `activate` call

### ISSUE THREAD

**Issue by ziadsawalha** [anton-johansson/klarna-nodejs#10]
Also add `activate_reservation` with "deprecated" warning.

**Comment by anton-johansson** [comment]
Do we really need the deprecated one? I assume equal things can be done with just `activate`?

**Comment by ziadsawalha** [comment]
Right, most of the functionality can be done via `update` and `activate`, but we plan to add it to support a corner case we're working on. We plan to implement that, so the issue is mostly to announce our intent.

**Comment by anton-johansson** [comment]
Allright, fair enough! :)
